public/app/features/migrate-to-cloud/cloud/EmptyState/MigrationStepsPane.tsx

```diff
@@ -54,7 +54,7 @@ export const MigrationStepsPane = () => {
       <TextLink href="/connections/private-data-source-connections">
         {t('migrate-to-cloud.get-started.configure-pdc-link', 'Configure PDC for this stack')}
       </TextLink>
-      <TextLink href="https://grafana.com/docs/grafana-cloud/account-management/migration-guide">
+      <TextLink href="/a/grafana-selfservetutorials-app/migrate-oss">
```

Comment:
I think the first `/` may not be needed @macabu, but please test first!

Reply:
```suggestion
<TextLink href="a/grafana-selfservetutorials-app/migrate-oss">
```

Reply:
@danielkenlee It works as-is 🙂